packages/react/src/jsx/ReactJSXElement.js

```diff
@@ -19,9 +19,9 @@ const RESERVED_PROPS = {
   __source: true,
 };
 
-let specialPropKeyWarningShown;
-let specialPropRefWarningShown;
-let didWarnAboutStringRefs;
+let specialPropKeyWarningShown,
+  specialPropRefWarningShown,
+  didWarnAboutStringRefs;
```

Comment:
This change seems purely subjective. Some may prefer one way, others another. We usually don't accept merge changes to the code.

Reply:
get, thank you for your reply.
I am sorry to bother your time, my intention just wants the React code better.🤣

Reply:
No worries. It's okay!

Reply:
hahaha, But another change I hope that can make changes in subsequent, because it does make a function call without necessary.